src/main/java/com/datastax/cdm/job/SplitPartitions.java

```diff
@@ -24,16 +24,17 @@
 import java.io.*;
 import java.math.BigInteger;
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
 public class SplitPartitions {
 
     public static Logger logger = LoggerFactory.getLogger(SplitPartitions.class.getName());
+    private static final int MAX_NUM_PARTS_FOR_PARTITION_FILE = 10;
```

Comment:
Sorry just a thought popped up now. Should we keep this setting with an unexposed configurable variable so we don't have to change the code in case we need to try out higher `numParts` in future?